src/Indice.Features.Cases.AspNetCore/Models/SendMessageRequest.cs

```diff
@@ -0,0 +1,17 @@
+﻿using Microsoft.AspNetCore.Http;
+
+namespace Indice.Features.Cases.Models;
+
+/// <summary>The request to insert a new message to a case.</summary>
+internal class SendMessageRequest
```

Comment:
έχε υπόψιν σου πως το "Message" ως οντοτητα στο case το έχουμε μοντελοποιήσει να περιλαμβάνει οποιαδήποτε νέο [ comment, checkpoint change, προσθηκη attachment].

Δεν λέω οτι είναι οτι καλύτερο, αλλά αυτό εδώ τώρα υποστηρίζει ενα απο τα 3 που κάνει. Ίσως μπορούμε να το δούμε σαν την αρχή να σπάσουν οι λειτουργίες και οταν θέλεις comment να κάνεις comment, οταν θες checkpoint να κάνεις checkpoint κ.ο.κ

@dkarkanas δες και εσύ

Reply:
Από οτι καταλαβα, θελουν να υλοποιήσουμε την chat δυνατότητα οπότε νομζω ειναι οκ να παει με το 1/3. Νομίζω ήδη "σπαστά" τα έχουμε και σε αλλα endpoints (πχ AddCaseData ή AddAttachment) ώστε να είναι πιο κατανοητό το swagger.

Το μόνο που θα άλλαζα είαι τα ονόματα, και κάνω τώρα suggestions

Reply:
Ναι chat λογικη θέλουμε όπως λέει και ο Δημήτρης, η οποία όμως δεν υποστηρίζεται. 
Ειδα όλα τα endpoints που βάζουν "Message" αλλα κανένα δεν βάζει μόνο text μηνύματα για να μας καλύψει.

Reply:
2 μικρές λεπτομέρειες ακόμα και κλείνει

- Μετονόμασε και την κλάση σε `SendCommentRequest` για να είμαστε safe, να μην εχει την λέξη Message και μας μπερδεύει.
- Πρόσθεσε μία εγγραφή στο changelog ότι add admin endpoint for posting new comments to a case φάση.